static/js/browser/statvar_group_node.tsx

```diff
@@ -150,6 +175,35 @@ export class StatVarGroupNode extends React.Component<
     );
   }
 
+  private fetchData(): void {
+    if (this.state.dataFetched) {
+      return;
+    }
+    // stat var (group) dcid count contain "&", need to encode here.
```

Comment:
Not a documented list :(.  The special chars are generally:  /_-.&

It is interesting why the this & comes about.  Happens in cases where we effectively concatenate two enums in a single V value. That is because P values in SV cannot be repeated, and we don't want to define new enums just to represent combinations.

Reply:
Thanks for the explanation, the example is like "dc/g/Person_Citizenship-NotAUSCitizen_CorrectionalFacilityOperator-StateOperated&FederallyOperated&PrivatelyOperated"

Reply:
If the `&` is a new addition with SVG, we'll need to test it with our other pages, eg browser. We have not had to worry about URL search special chars before, so the encoding might need to be propagated.

Could we use `_and_` or `,` instead?

Reply:
That's a good point, any page with this stat var will not show properly..

Reply:
Good points.

1. At the moment, there are only 2 SV IDs with '&' in it (https://screenshot.googleplex.com/83cTHghienxMjUS).
2. That SV node [shows up](https://datacommons.org/browser/USStateQuarterlyIndustryGDP_NAICS_311_316&322_326), but naics PV is missing looks like. Also, oddly there are no SVObs for these SVs (https://screenshot.googleplex.com/6bySnwSjjfvaHuD). So I think nothing rn is too broken.
3. And yes, we can remove '&' from the SVG ID.

`,` is a delimiter too. but `_And_` works.